pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -87,7 +87,7 @@ var _ = Describe("Qemu agent poller", func() {
 
 	Context("PollerWorker", func() {
 		It("executes the agent commands at least once", func() {
-			const interval = 1
+			const interval = 2
```

Comment:
Please keep an identical coverage, this is checking a different code path in he production code.
There should be no need to change any existing test except adding the initial-interval as an input.